__snapshots__/wallet.js

```diff
@@ -22,3 +22,27 @@ exports['wallet create 1'] = {
   },
   "publishers": {}
 }
+
+exports['wallet recovery succeeds with a valid key 1'] = {
```

Comment:
We are missing some fields

```
{
	"paymentId": "f802a9ef-b025-4d94-a811-3ab52e043701",
	"addresses": {
		"BAT": "0x6f2d5Bc4145e5797993A81733A0a230a799Dfbe1",
		"BTC": "1GX5eQcCEpUZx4CQwf1sH42DZa1dcQE9Bp",
		"CARD_ID": "a6cc6071-a036-464d-95a1-4d17d9bacbf1",
		"ETH": "0x6f2d5Bc4145e5797993A81733A0a230a799Dfbe1",
		"LTC": "LWvKc9JgnA7TCsTEfgNxLmQv4qVhBPwJbF"
	},
	"unconfirmed": 0,
	"hasBitcoinHandler": false,
	"bravery": {
		"setting": "adFree",
		"days": 30,
		"fee": {
			"currency": "BAT",
			"amount": 7.5
		}
	},
	"contributionAmount": 10,
	"created": true,
	"passphrase": "crucial effort away vintage dilemma real title local tomato snow unhappy place know west dawn drift pelican swim beyond swarm clump ski suggest elevator",
	"reconcileFrequency": 30,
	"monthlyAmounts": [
		5,
		7.5,
		10,
		12.5,
		15,
		17.5,
		20,
		25,
		50
	],
	"converted": 0,
	"rates": {
		"BTC": "0.00003910",
		"ETH": 0.0004838222334130681,
		"LTC": 0.002425064681012179,
		"USD": 0.3074796263,
		"EUR": 0.2613312391071382
	},
	"probi": 0,
	"reconcileStamp": 1529739810578,
	"transactions": [],
	"currentRate": 0.3074796263,
	"creating": false,
	"walletQR": {
		"BAT": "data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAM0AAADNCAAAAAA+16u1AAACJklEQVR42u3bS26DQBQEQO5/6WQZxZJx93zATopVJLAyxeLp/Ti+/tJ10NDQ0NDQ0NDQ0NDQ0NDQzGqO19fPc09/8fBI+j+qE9DQ0GzUPA8cv+9WpPMTBS/hxSuioaFZrQnCVB+XHu6ea7pASUNDc60mTjjCuzQ0NJ+rSZOfICjS0NC8lSawBqccjHMXV2s0NDTTvc7Vf93buaWh+deaeMxY90TTt3PHFJeGhub8lEHFMTjiSCPZyjyNhoYmjWnpwKJ6JPCn7RQaGpqNmioGBUOMLdMSGhqajZr+bBU47Y6mSQ0NDc0ezeCacrDqFOxUDYxRaGho1mimFxqC1KgKZ1P1DQ0Nzeb6ZnXTo9qgpqGhuUQTDECCFsZ5JJtegqShoVmpqfaXg8XIat1h9xSXhoamimlpVRNkNtU2RRC/aGhoNmr67mMKrrKnRXNPGhqaK3qdg++kmqjS0NDcoak2o6viJZ2jzm8/0NDQDGpmIPtiJA0NzSWa6c5HOthI+6Q0NDTvokmHE/0XFdVUZXSvk4aGZjCzSc8xPcWcXoSioaG5N6YFiUmVEKUjVBoamuWatPsY1DLTY9BFEw8aGprpuedglzJNZdb0RGloaJZr0s8WghBXtTCWV2s0NDRbNGlfpA+AgzkODQ3NbZqgUBl8EzQ0NPdqqvqm2nBMh59d85WGhmahpjpM/wVEHy2D52hoaBZpPvaioaGhoaGhoaGhoaGhoaFprm8jCHWHgt3taQAAAABJRU5ErkJggg==",
		"BTC": "data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAALkAAAC5CAAAAABRxsGAAAABwklEQVR42u3aQXKDMAwFUO5/abruDKhfNm1q/LxKQsAvmUGWLI5z1XGQk5OTk5OTk5OTk5OTv0V+/Dzq064O1K8GZiMn31p+f1N/P1pPdwWcmY2cnPz+vr+ND+m5Qby5nY2cnHxUfgUMUgVycvLflrds9VtycvIn8/OrSeoA818qC3Ly5eXBrtODr/54d46cfGV53KopY0s98Wc7XOTkK8uD+z5Y3+uQ1JqDnJw8lgepQhqSWv8JOTn5ObgnNfirp34/OfnW8qB47veO6o2udAOanJw8CQNpVd0KSYGAnJz8aKXmQQOp7u4+2cklJ99BnubOwdH6orV8vrIgJ3+5PF3zW5+1LkVOTh6vwkFBnZ4RNIWn8hZy8g3laS4ebHQFW2jk5OQD+flgARxk70/mLeTkO8hbXZ/W8xJ1CT4fW8jJXy5vPZk/mJD30eTke8vT0a+0Z3J2cnLyM40jQS0dBKeZJJ2cfG95HVH6ndfBVH++siAnf688Xd/r2NJa6edXf3Jy8n6USb/yUJ+InJx8unhuhSlycvJefl7n3TNPbCQMcvKt5Wk7qNWh7T/XSE5OvuQgJycnJycnJycnJycnX3F8AVZ3tS44RtqnAAAAAElFTkSuQmCC",
		"ETH": "data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAM0AAADNCAAAAAA+16u1AAACJklEQVR42u3bS26DQBQEQO5/6WQZxZJx93zATopVJLAyxeLp/Ti+/tJ10NDQ0NDQ0NDQ0NDQ0NDQzGqO19fPc09/8fBI+j+qE9DQ0GzUPA8cv+9WpPMTBS/hxSuioaFZrQnCVB+XHu6ea7pASUNDc60mTjjCuzQ0NJ+rSZOfICjS0NC8lSawBqccjHMXV2s0NDTTvc7Vf93buaWh+deaeMxY90TTt3PHFJeGhub8lEHFMTjiSCPZyjyNhoYmjWnpwKJ6JPCn7RQaGpqNmioGBUOMLdMSGhqajZr+bBU47Y6mSQ0NDc0ezeCacrDqFOxUDYxRaGho1mimFxqC1KgKZ1P1DQ0Nzeb6ZnXTo9qgpqGhuUQTDECCFsZ5JJtegqShoVmpqfaXg8XIat1h9xSXhoamimlpVRNkNtU2RRC/aGhoNmr67mMKrrKnRXNPGhqaK3qdg++kmqjS0NDcoak2o6viJZ2jzm8/0NDQDGpmIPtiJA0NzSWa6c5HOthI+6Q0NDTvokmHE/0XFdVUZXSvk4aGZjCzSc8xPcWcXoSioaG5N6YFiUmVEKUjVBoamuWatPsY1DLTY9BFEw8aGprpuedglzJNZdb0RGloaJZr0s8WghBXtTCWV2s0NDRbNGlfpA+AgzkODQ3NbZqgUBl8EzQ0NPdqqvqm2nBMh59d85WGhmahpjpM/wVEHy2D52hoaBZpPvaioaGhoaGhoaGhoaGhoaFprm8jCHWHgt3taQAAAABJRU5ErkJggg==",
		"LTC": "data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAM0AAADNCAAAAAA+16u1AAACMUlEQVR42u3aS46DQAwFQO5/6Zn1SAHe6w9JRsUqSgRxsbDcto+f/3QdNDQ0NDQ0NDQ0NDQ0NDQ0s5rj/np1x+kDgu+mI6ChoVmuOU8cf4N5FUJw701yGnsKDQ3Ncs1pIgo0N+nnMlelEdDQ0HyUZvBeGhqa79WkAQYnmJkIaGhoNmrSX69JVep622mNhoam7zRu/vRw55aGhmZgzHj/T8F3QYv0kSkuDQ3N4MSjj/f6juBXGhqa3ZqZYUdQBlWv7bSnQkND84gmSDqDY4p0glIVPzQ0NMs16Uji2jCY8RZNPGhoaKY1awYWaRYM8hwNDc1uzXT38TpDDVZPU9sPNDQ00ztQaa8zqGyqhmc1GaGhoVmkSTuNQfGTVicPn29oaGiqyiYYQg4UIcFGU53YaGhoVmriaqKZbqQ9kLQ+oqGh2aOpOhVpPgyirEooGhqajZpq6alKTv1oJU1nNDQ0KzXpyaSvYtakLhoamt2aYGkxnVDMzEcHnkdDQ7NZk5YeQQ+z6q4sqtNoaGj25bR+YJEWNd17oqGhWahJN5TTVeiZRBlPX2hoaBbuDiZNhfvcV604X78JGhqa5zT9uSXIZOmpJn0dNDQ0uzVVg7IqTKpQl5/WaGhoKk265hjEERQrQeQ0NDSfpwmqkypbXtNviiQaGpo3aPom6OBi5HMZmoaGpj/fVG2NmfPSzaNoaGgWaqYnkVWvsxqA0NDQ7NZ87UVDQ0NDQ0NDQ0NDQ0NDQ9Ncv4YXQ7m7uLQZAAAAAElFTkSuQmCC"
	},
	"balance": 0
}
```

This is what I get if I create new wallet in [`ledger`, `info`]. We need to have the same data here as well